macos/Sources/App/macOS/AppDelegate.swift

```diff
@@ -523,6 +523,9 @@ class AppDelegate: NSObject,
         // this is called as part of application launch it can deadlock with an internal
         // AppKit mutex on the appearance.
         DispatchQueue.main.async { self.syncAppearance(config: config) }
+        
+        // Decide whether to hide/unhide app from dock and cmd-tab
```

Comment:
@xalbd  // Decide whether to hide/unhide app from dock and app switcher

Reply:
@xalbd Also, why do you call it cmd tab and not its official name "app switcher"?